### PR TITLE
refactor/842

### LINKED ISSUE #842: Update KBOS to AIRAC 1801 and add airport guide
LBSTA5 departure was renamed to LBSTA6 (procedure remains unchanged)

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #842 

-Renames LBSTA departure
-Adds airport guide
-Improves video map

### CODE REVIEW

File: assets/airports/kbos.json [hunk 1]
```diff
@@ -574,7 +574,7 @@
             "icao": "WYLLY3",
             "name": "Willy Three",
             "rwy": {
-                "27": ["_BOS261002", "WYLYY", "CYOTI"]
+                "KBOS27": ["_BOS261002", "WYLYY", "CYOTI"]
             },
             "body": [],
             "exitPoints": {
```
Comment (indianbhaji) [inline assets/airports/kbos.json]: Extra line after `SSOXS`

File: documentation/airport-guides/kbos.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+### About this facility
+Boston TRACON (A90), Merrimack, New Hampshire, USA
+658,524 annual operations in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+Located almost 40 miles Northwest of KBOS, this consolidated approach control facility is one of six large TRACON's in the United States. It controls a large portion of the states of Massachusetts and New Hampshire
```
Comment (indianbhaji) [inline documentation/airport-guides/kbos.md]: TRACONs, assuming you want the plural
's is possessive (ie. Kim's car)

Full stop/period (whatever you call it) after New Hampshire

File: documentation/airport-guides/kbos.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+### Airspace
+Due to it's sheer size, the A90 TRACON is divided into two areas; Boston and Manchester. However, only the Boston area is available in the sim; It extends 40nm around KBOS up to 14,000ft. Class B airspace exists with a ceiling of 7,000ft over the Boston metropolitan area.
```
Comment (indianbhaji) [inline documentation/airport-guides/kbos.md]: it's = it is
Should be its

File: documentation/airport-guides/kbos.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+13.9nm Northeast of KBOS
+
+Operated by the U.S. Navy during World War II, Beverly airport is now a small regional airfield used primarily by general aviation.
+
+#### KLWM
+Lawrence Municipal Airport  
+40,148 annual operations  
+21.9nm North of KBOS
+
+Located beside Lake Cochichewick, Lawrence Airport is a small municipal airfield used for general and private aviation.
```
Comment (indianbhaji) [inline documentation/airport-guides/kbos.md]: Could add a section with additional resources, linking to the FAA Charts

File: assets/airports/kbos.json [hunk 5]
```diff
@@ -8542,173 +3767,629 @@
...
+            ["N42d07m46.154", "W70d41m34.481", "N42d07m01.194", "W70d40m35.423"],
+            ["N42d06m21.844", "W70d39m42.992", "N42d05m35.213", "W70d38m41.334"],
+            ["N42d04m55.520", "W70d37m48.945", "N42d04m09.864", "W70d36m48.653"],
+            ["N42d03m29.494", "W70d35m55.423", "N42d02m45.807", "W70d34m57.815"],
+            ["N42d02m04.099", "W70d34m02.866", "N42d01m19.734", "W70d33m04.423"],
+            ["N42d20m34.435", "W70d58m35.792", "N42d19m49.953", "W70d57m36.333"],
+            ["N42d10m24.230", "W70d45m58.925", "N42d11m09.220", "W70d44m57.639"],
+            ["N42d10m40.286", "W70d45m20.340", "N42d09m54.697", "W70d44m19.867"],
+            ["N42d14m11.031", "W70d51m05.660", "N42d14m56.066", "W70d50m04.374"],
+["N42d16m16.977", "W70d52m52.671", "N42d16m10.710", "W70d52m44.701"],
```
Comment (indianbhaji) [inline assets/airports/kbos.json]: Indent these two lines please (4391)

File: assets/airports/kbos.json [hunk 6]
```diff
@@ -848,7 +848,7 @@
             "origin": "kbos",
             "destination": "",
             "category": "departure",
-            "route": "KBOS.HYLND5.HYLND..CAM",
+            "route": "KBOS22R.HYLND5.HYLND..CAM",
```
[the remark this answers is not in the document]
Comment (Fechulo) [reply assets/airports/kbos.json]: Although it's shown on the charts. it's not a mandatory fix. For example [SKV7687](https://flightaware.com/live/flight/SKV7687/history/20180212/2330Z/KBOS/CYYZ), flies the route `KBOS HYLND5 HYLND CAM GONZZ WOZEE LINNG8 CYYZ`

File: assets/airports/kbos.json [hunk 7]
```diff
@@ -996,9 +996,9 @@
...
             "destination": "kbos",
             "category": "arrival",
-            "route": "AJJAY.OOSHN5.KBOS",
+            "route": "AJJAY.OOSHN5.KBOS22L",
             "altitude": [30000, 36000],
-            "speed": 450,
+            "speed": 320,
             "method": "cyclic",
             "rate": 7,
             "period": 60,
```
[the remark this answers is not in the document]
Comment (Fechulo) [reply assets/airports/kbos.json]: cyclic
Variation is a required key for cyclic spawnPatterns, so there _should_ be no warning

File: assets/airports/kbos.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (indianbhaji) [reply assets/airports/kbos.json]: No problem 👍

File: assets/airports/kbos.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (indianbhaji) [reply assets/airports/kbos.json]: Will double check this

**Comment by erikquinn** [comment]
@n8rzz We have 231 conflicts with `package-lock.json`, which I don't know how it works. How should we handle this?

**Comment by n8rzz** [comment]
remove the local version and pull from `develop`.  Or, remove local version `npm install` then commit changes.

Since our packages haven't changed in some time, I'd go with the first option.

**Pull request merged**